ui/app/adapters/job-version.js

```diff
@@ -0,0 +1,18 @@
+import ApplicationAdapter from './application';
+import addToPath from 'nomad-ui/utils/add-to-path';
+
+export default class JobVersionAdapter extends ApplicationAdapter {
+  revertTo(jobVersion) {
+    const jobAdapter = this.store.adapterFor('job');
+
+    const url = addToPath(jobAdapter.urlForFindRecord(jobVersion.get('job.id'), 'job'), '/revert');
+    const [jobName] = JSON.parse(jobVersion.get('job.id'));
+
+    return this.ajax(url, 'POST', {
+      data: {
+        JobID: jobName,
+        JobVersion: jobVersion.number,
+      },
+    });
```

Comment:
I forget how this URL building stuff works. Is this going to append the namespace as a query param?

Reply:
ya, I did manually test that but it makes sense to automate, how does c30917f look for that?

Reply:
Looks good!